feat: support raw features (#1386)
kubernetes-sigs/node-feature-discovery: docs/usage/customization-guide.md

> tag is only evaluated in each re-discovery period, and the expiration of
> node labels is not tracked.

To exclude specific features from the `local.feature` Feature, you can use the

> `# +no-feature` directive. The same applies to `local.label` Feature where you can
> exclude some features using the `# +no-label` directive.

Reviewer: We need to also mention that no labels are generated with `+noLabel`

```suggestion
`# +no-feature` directive. The `# +no-label` directive causes the feature to be excluded from the
`local.label` Feature and a node label not to be generated.
```

or smth

Note: now that we have already three directives we could add e.g. a table to list them all. But this can be left as a future enhancement.

Considering the following file:

```plaintext
# +no-feature
label-only=value

my-feature=value

foo=bar
# +no-label
foo=baz
```

Processing the above file would result in the following Features:

```yaml
local.features:
  foo: baz
  my-feature: value
local.labels:
  label-only: value
  my-feature: value
```

> 

Reviewer: Suggest an addition to mention the node labels, too (the indentation below is wrong to get the formatting right...)

```suggestion
    and the following labels added to the Node:
    
    ```
    feature.node.kubernetes.io/label-only=value
    feature.node.kubernetes.io/my-feature=value
    ```
```

|                  |              | **`major`** | int     | First component of the kernel version (e.g. ‘4')
|                  |              | **`minor`** | int     | Second component of the kernel version (e.g. ‘5')
|                  |              | **`revision`** | int  | Third component of the kernel version (e.g. ‘6')

> | **`local.label`** | attribute   |           |           | Labels feature files and hooks, i.e. labels from the [*local* feature source](#local-feature-source)

Reviewer: ```suggestion
| **`local.label`** | attribute   |           |           | Labels from feature files and hooks, i.e. labels from the [*local* feature source](#local-feature-source)
```
